src/components/Books.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Form from './Form';
+
+const Books = ({ title, author }) => (
+  <>
+    <div>
+      <h2 className="book">{title}</h2>
+      <p className="by">by</p>
+      <p className="author">{author}</p>
+      <button type="button" className="remove-btn">Delete</button>
+    </div>
```

Comment:
- [ ] Please ensure that your `<Books>` component, contains the following components:

  * `<BookList>`
    * Renders a list of books using the `<Book>` component
  * `<Book>`
    * Takes the book data as a prop and renders the title and the author
    * Has a button for removing books
      * the button wouldn't remove the books *yet*, this is just a placeholder for now

Reply:
@Owusu-Desmond, in the project requirements, microverse did not mention interactivity, it is just a set up(initialization).

Reply:
@Owusu-Desmond, take a look ![Screenshot_20230929-192113](https://github.com/yayaismaail/bookstore/assets/131773841/62bfe30f-8645-4d7e-a427-b3a0fb87f65f)

Reply:
- [ ] Just as the previous reviewer mentioned, there should be a component called `BookList` which renders another component called `Book` with props, so what you can do here is two you can create a new component, call it BookList or anything you like, create an array of books in it, map through the array and render this particular component, then import the BookList in you App.js, or you turn this into the booklist component and the create a new Book component that will take props and render it here